### PR TITLE
Backport proutes improvements from 1.6 to 1.5

### PR DESCRIPTION
**Pull request by sontek**
See https://github.com/Pylons/pyramid/pull/1488 for more info.

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -4,6 +4,11 @@ Next release
 - Fixed a failing unittest caused by differing mimetypes on various
   OS platforms. See https://github.com/Pylons/pyramid/issues/1405
 
+- Overall improvments for the ``proutes`` command. Added ``--format`` and
+  ``--glob`` arguments to the command, introduced the ``method``
+  column for displaying available request methods, and improved the ``view``
+  output by showing the module instead of just ``__repr__``.
```
Comment (mmerickel) [inline CHANGES.txt]: Link the PR please!

Comment (sontek) [inline CHANGES.txt]: Yeah, I didn't know if I should link #1488 or this specific backport one, what do you think?

Comment (mmerickel) [inline CHANGES.txt]: I think you should link this one. The original is linked from this PR already.

**Comment by sontek** [comment]
@mmerickel link added

**Pull request merged**